Vagrantfile

```diff
@@ -9,6 +9,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.network :forwarded_port, guest: 8282, host: 8282
     config.vm.network :forwarded_port, guest: 80, host: 8000
     config.vm.network :forwarded_port, guest: 4443, host: 4443
+    config.vm.network :forwarded_port, guest: 777, host: 777
```

Comment:
hm. y'know.... you might be right.  if 'ssl_client_certificate' and 'ssl_verify_client' can be put into a 'location' directive in nginx, that might work.

I think I confused myself into thinking that the server cert also needed to be signed by our internal CA